compatibility/versions.bzl

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Would be good to include a comment that this is a generated file and shouldn't be updated manually and point to the `update-versions.sh` script.